util/src/kvdb.rs

```diff
@@ -598,4 +651,12 @@ mod tests {
 		let _ = Database::open_default(path.as_path().to_str().unwrap()).unwrap();
 		test_db(&DatabaseConfig::default());
 	}
+
+	#[test]
+	fn auto_compaction() {
```

Comment:
hm, I guess so. seems like a strange thing to test on any platform where it isn't relevant though.